packages/old-import/src/widgets/options.ts

```diff
@@ -103,6 +103,12 @@ const optionMapping: OptionMapping = {
   indexerManager: {
     openIndexerSiteInNewTab: (oldOptions) => oldOptions.openIndexerSiteInNewTab,
   },
+  healthMonitoring: {
+    cpu: (oldOptions) => oldOptions.cpu,
+    memory: (oldOptions) => oldOptions.memory,
+    fahrenheit: (oldOptions) => oldOptions.fahrenheit,
+    fileSystem: (oldOptions) => oldOptions.fileSystem,
+  },
```

Comment:
We had more options in the old Homarr. Will you implement them in a follow up PR? Or are they no longer required?

Reply:
The other options from old Homarr is for Proxmox integration. So it will take care when Proxmox integration will created